modules/core/src/main/scala/api/messages/configuration/CreateLegalBasePayload.scala

```diff
@@ -29,9 +30,19 @@ object ScopePayload {
 }
 
 case class CreateLegalBasePayload(
+    @description("type of the legal base")
+    @encodedExample("CONSENT")
     lbType: LegalBaseTerms,
+    @description("legal base name")
+    @encodedExample("Contact form")
     name: Option[String],
+    @description("legal base description")
+    @encodedExample("Collection of the contact data for advertising")
     description: Option[String],
+    @description("legal base description")
```

Comment:
I believe this is a copy-paste typo? :slightly_smiling_face:

Reply:
Fixed